docs/src/algorithms/discretization.md

```diff
@@ -5,3 +5,53 @@ discretize
 DiscretizationMethod
 FIST
 ```
+# FIST  
+Examples for the FIST algorithm. 
+
+### Usage  
+  
+```@example overview
+# Import required Libraries
+using GeoStats
+using Plots; gr(size=(300,300))
+
+# Function to read Polygon from given values
+function readpoly(T, fname) 
+  open(fname, "r") do f
+    # read outer chain
+    n = parse(Int, readline(f))
+    outer = map(1:n) do _
+      coords = readline(f)
+      x, y = parse.(T, split(coords))
+      Point(x, y)
+    end
+
+    # read inner chains
+    inners = []
+    while !eof(f)
+      n = parse(Int, readline(f))
+      inner = map(1:n) do _
+        coords = readline(f)
+        x, y = parse.(T, split(coords))
+        Point(x, y)
+      end
+      push!(inners, inner)
+    end
+
+    # return polygonal area
+    PolyArea(outer, inners)
+  end
+end
```

Comment:
What about manually copying the lines of the file in this example? What I mean is that we don't need to bother the reader with a complex IO function. Instead we could provide a single line like:

```julia
polyarea = PolyArea([(1.0,2.5), (1.3,12.0), ...]) # big list of points manually copied from file
```

And plot it directly so that users can see it with their own eyes. That way we also avoid a dependency on a test file.

Reply:
Working on this!